infra-pipeline/pipeline/inputs.tf

```diff
@@ -49,6 +49,25 @@ variable "tf_root" {
   description = "The location (source repo and path) to run terraform commands"
 }
 
+variable "notification_topics" {
+  type = map(object({
+    emails = set(string)
```

Comment:
I recommend using `map(object({protocol = string, subscriptions = set(string)}))`

Reply:
And maybe adding a comment with the link to https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sns_topic_subscription#protocol-support